v3/newrelic/internal_17_test.go

```diff
@@ -18,6 +19,37 @@ func myErrorHandler(w http.ResponseWriter, req *http.Request) {
 	txn.NoticeError(myError{})
 }
 
+func TestWrapHandleFastHTTPFunc(t *testing.T) {
+	app := testApp(nil, ConfigDistributedTracerEnabled(true), t)
+
+	// Define your handler
+	handler := func(ctx *fasthttp.RequestCtx) {
+		ctx.WriteString("Hello World")
+	}
+
+	_, wrappedHandler := WrapHandleFuncFastHTTP(app.Application, "/hello", handler)
+
+	if wrappedHandler == nil {
+		t.Error("Error when creating a wrapped handler")
+	}
+	ctx := &fasthttp.RequestCtx{}
+	ctx.Request.Header.SetMethod("GET")
+	ctx.Request.SetRequestURI("/hello")
+	wrappedHandler(ctx)
+
+	app.ExpectMetrics(t, []internal.WantMetric{
```

Comment:
I noticed the wrapHandle test also creates and expects an error. Can we do that for the tests for fasthttp as well?

Reply:
there's an example of noticing an error in the fasthttp-server example :) I don't think it should be too difficult to add that into the internal testing as well.